act/virtualNetwork.tf

```diff
@@ -86,25 +86,55 @@ resource "azurerm_storage_account" "my_storage" {
   location                 = azurerm_resource_group.resource_group.location
   account_tier             = "Standard"
   account_replication_type = "LRS"
+
+  queue_properties {
+    logging {
+      write                 = true
+      read                  = true
+      version               = "1.0"
+      retention_policy_days = 365
+      delete                = true
+    }
+    hour_metrics {
+      enabled               = true
+      version               = "1.0"
+      include_apis          = true
+      retention_policy_days = 365
+    }
```

Comment:
**Accurics remediation**
 
**[LOW]: Ensure that authentication feature is enabled for Azure Function App**
 
**Vulnerability**
Azure Function App do not have authentication feature enabled, this may lead to unauthorized access/data leak. 
 
**Remediation**
Azure App Service provides built-in authentication and authorization capabilities (sometimes referred to as 'Easy Auth'), so you can sign in users and access data by writing minimal or no code in your web app, RESTful API, and mobile back end, and Azure Function App based applications. 

source: accurics-beta